### DISCUSSION TITLE
What should be the correct path to the photo when I upload my project to the VPS server

### DISCUSSION THREAD

**Discussion by sadness-mugetsu** [SixLabors/ImageSharp#1359] [Q&A] [answered]
I uploaded my bot on VPS server publish folder is responsible for the publication and compilation of my project also threw in a folder publish a folder Image I pointed out the way to photography 
`var background = SixLabors.ImageSharp.Image.Load("/root/publish/Images/my photo");` very strange when I write the command it does not processed and comes out in the console an exception, tell me what you need to specify the correct path to the photo when you publish your project

**Comment by JimBobSquarePants** [comment, marked as answer]
You’re path needs to be the absolute path to the image source.
